app/controllers/api/account.php

```diff
@@ -44,6 +44,96 @@
 $oauthDefaultSuccess = '/auth/oauth2/success';
 $oauthDefaultFailure = '/auth/oauth2/failure';
 
+App::post('/v1/account/invite')
+    ->desc('Create Account using an invite code')
+    ->groups(['api', 'account', 'auth'])
```

Comment:
How do we skip the auth protection we have in the special auth init method?

Referring to these checks: 
https://github.com/appwrite/appwrite/blob/master/app/controllers/shared/api.php#L267-L268

Reply:
I don't see how its a problem ? 
Also for the console project these checks are skipped anyways 
https://github.com/appwrite/appwrite/blob/master/app/controllers/shared/api.php#L279

Reply:
Got it what about the whitelist check? how is that being skipped?